gen/flights/flightplan.py

```diff
@@ -1195,39 +1189,79 @@ def generate_strike(self, flight: Flight) -> StrikeFlightPlan:
             flight, location, FlightWaypointType.INGRESS_STRIKE, targets
         )
 
-    def generate_aewc(self, flight: Flight) -> AwacsFlightPlan:
-        """Generate a AWACS flight at a given location.
+    def generate_aewc_racetrack(self, flight: Flight) -> AwacsFlightPlan:
+        """Generate a AWACS flight in racetrack orbit.
 
         Args:
             flight: The flight to generate the flight plan for.
         """
+
+        racetrack_half_distance = Distance.from_nautical_miles(20).meters
```

Comment:
```suggestion
        racetrack_half_distance = Distance.from_nautical_miles(30).meters
```

AWACS should probably have longer legs to turn less often.